app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/evaluateCloudFormationChangeSetExecution.service.ts

```diff
@@ -0,0 +1,47 @@
+import { IPromise, module } from 'angular';
+import { EXECUTION_SERVICE, ExecutionService } from '@spinnaker/core';
+import { IExecution, IExecutionStage } from 'core/domain';
+import { Application } from 'core/application';
+
+export class EvaluateCloudFormationChangeSetExecutionService {
+  public static $inject = ['executionService'];
+  constructor(private executionService: ExecutionService) {}
+
+  private clearPipelineColor(): void {
+    if (
+      typeof document.getElementsByClassName(
+        'execution-marker-running stage-type-deploycloudformation stage-type-deploycloudformation-ask',
+      )[0] !== 'undefined'
+    ) {
+      document
+        .getElementsByClassName(
+          'execution-marker-running stage-type-deploycloudformation stage-type-deploycloudformation-ask',
+        )[0]
+        .classList.remove('stage-type-deploycloudformation-ask');
+    }
```

Comment:
See my comments on `evaluateCloudFormationChangeSetExecutionMarkerIcon.tsx` regarding this conditional block.

Reply:
Class changes aren't needed anymore after add new style to execution marker